### PR TITLE
Update hello example to ES6

### PR DESCRIPTION
**Pull request by camstuart**
- Updated `package.json` an npm script to launch the application
- Updated `README.md` with launch instructions

### CODE REVIEW

File: examples/hello/README.md [hunk 1]
```diff
@@ -3,3 +3,21 @@
 Our interactive Hello World. Also a good template for your first own app. [This article](http://eclipsesource.com/blogs/2015/02/20/tabris-js-examples-hello-world/) discusses the example in detail.
 
 ![Hello world](https://tabrisjs.com/assets/public-content/img/examples/hello-world-android-ios.png)
+
+## Running this example
```
Comment (cpetrov) [inline examples/hello/README.md]: Can you please move the _Running this example_ part to examples/README.md and adjust it accordingly? It applies to all examples.

**Comment by camstuart** [comment]
@cpetrov Amended the commit as requested

File: examples/hello/README.md (hunk 1 above)
Comment (camstuart) [inline examples/hello/README.md]: Done

**Comment by ralfstx** [comment]
Thanks @camstuart for the contribution!

**Pull request merged**